app/src/test/java/gov/va/vro/controller/VroControllerTest.java

```diff
@@ -249,6 +249,7 @@ void generatePdfInvalidInput() {
     generatePdf.setClaimSubmissionId("1234");
     generatePdf.setVeteranInfo(new VeteranInfo());
     generatePdf.setEvidence(new AbdEvidence());
+    generatePdf.setPdfTemplate("v1");
```

Comment:
@I am assuming v1 should be default? No. This end point should will not be used in mas workflow at all so I would leave older test as is.

Reply:
MAS is using "generate PDF"

Reply:
@dimitri-amida Why are we using the end point? I thought Camel was our integration pattern.

Reply:
@au2 I am not sure what you mean by the "end point" in this context. We are calling the Generate PDF route (via camel) to generate the final PDF, so any changes to the template and response/request objects ultimately affects MAS.

Reply:
I meant the REST end point; this being a controller test I am assuming that is what it tests. If you are just calling the Camel routes in MAS integration all is good. If you are calling the REST end point then I do not undertstand.